src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -185,6 +199,38 @@ export class SentryAppExternalIssueActions extends React.Component {
     this.setState({action: 'create'});
   };
 
+  deleteIssue = () => {
+    const {api, group} = this.props;
+    const {externalIssue} = this.state;
+    const url = `/issues/${group.id}/external-issues/${externalIssue.id}/`;
+
+    api.request(url, {
```

Comment:
Do we use this elsewhere? Would be nice to move this into actionCreator

Reply:
Even if we don't re-use it having the separation of UI rendering and API request creation is a good thing to have.

Reply:
This request, specifically, only happens here, but yea, +1 to moving API communication away from rendering. Will do.